### PR TITLE
[更新完毕] contributor_guide

### PR DESCRIPTION
**Pull request by ArkhamWJZ**
Related issue: #65

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/contributor_guide.po [hunk 1]
```diff
@@ -70,17 +71,19 @@ msgstr "加入我们的 `Taichi 论坛 <https://forum.taichi.graphics/>`_"
 
 #: ../../contributor_guide.rst:16
 msgid ""
-"Introduce Taichi to your friends or simply star `the project "
-"<https://github.com/taichi-dev/taichi>`_."
+"Introduce Taichi to your friends or simply star `the project <https://github.com/"
+"taichi-dev/taichi>`_."
 msgstr ""
 "向朋友们介绍 Taichi 或者在 GitHub 上星标一下 `Taichi <https://github.com/taichi-"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/contributor_guide.po]:
```suggestion
"向你的朋友们介绍 Taichi 或者直接在 GitHub 上星标 `Taichi <https://github.com/taichi-"
```

File: locales/zh_CN/LC_MESSAGES/contributor_guide.po [hunk 2]
```diff
@@ -70,17 +71,19 @@ msgstr "加入我们的 `Taichi 论坛 <https://forum.taichi.graphics/>`_"
...
 
 #: ../../contributor_guide.rst:17
 msgid ""
-"Typo fixes in the documentation, code or comments (please directly make a"
-" pull request for minor issues like these)"
-msgstr "修复文档，代码，注释中的拼写错误（这些小问题的话，请直接创建一个 PR 而不必开一个 issue）"
+"Typo fixes in the documentation, code or comments (please directly make a pull "
+"request for minor issues like these)"
+msgstr ""
+"修复文档，代码，注释中的拼写错误（这些小问题的话，请直接创建一个 PR 而不必开一个 "
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/contributor_guide.po]:
```suggestion
"修复文档，代码，注释中的拼写错误（像这样的小问题请直接创建一个 PR 而不必开一个 "
```

File: locales/zh_CN/LC_MESSAGES/contributor_guide.po [hunk 3]
```diff
@@ -595,146 +613,145 @@ msgstr "``[Bug]``: 修复 Bug;"
...
+"We do appreciate all kinds of contributions, yet we should not expose the title "
+"of every PR to end-users. Therefore the changelog will distinguish `what the "
+"user should know` from `what the developers are doing`. This is done by "
+"**capitalizing PR tags**:"
 msgstr ""
-"我们感谢所有的贡献，但是我们不应该把每一个 PR 的标题暴露给终端用户。因此，应该将变更日志分类成 `用户应该知道什么` 和 "
-"`开发人员正在做什么` 是必要的。而这是通过 **大写的 PR 标签** 区分的:"
+"我们感谢所有的贡献，但是我们不应该把每一个 PR 的标题暴露给终端用户。因此，应该将"
+"变更日志分类成 `用户应该知道什么` 和 `开发人员正在做什么` 是必要的。而这是通过 **"
+"大写的 PR 标签** 区分的:"
```
Comment (isdanni) [inline locales/zh_CN/LC_MESSAGES/contributor_guide.po]:
```suggestion
"大写 PR 标签** 区分的:"
```

Comment (isdanni) [reply locales/zh_CN/LC_MESSAGES/contributor_guide.po]: This should be a verb, meaning make it uppercase ;-)

**Review by isdanni — COMMENTED** [review]
LGTM！

File: locales/zh_CN/LC_MESSAGES/contributor_guide.po (hunk 3 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/contributor_guide.po]: That's very good advice!

**Review by isdanni — APPROVED** [review]

File: locales/zh_CN/LC_MESSAGES/contributor_guide.po [hunk 4]
```diff
@@ -154,33 +164,38 @@ msgstr "单元/集成测试是我们的好伙伴。"
...
+"research project. This usually means that some parts did not have the luxury to "
+"go through a solid design. While we are always trying to improve the code "
+"quality, it doesn't mean that the project is free from technical debts. Some "
+"places may be confusing or overly complicated. Whenever you spot one, you are "
+"more than welcome to shoot us a PR! :-)"
 msgstr ""
+"需要记住的一点是，Taichi 最初作为一个学术研究项目而诞生。通常情况下这意味着有些部"
+"件是没有奢侈到会经过稳固坚实的设计。虽然我们一直在努力提高代码质量，但并不意味着"
+"项目能没有技术负债。有些地方仍可能会让人感到困惑或过于复杂。无论何时你发现有这样"
+"的部分存在，都非常欢迎你给我们提出 PR！:-)"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/contributor_guide.po]:
```suggestion
"需要记住的一点是，Taichi 最初作为一个学术研究项目而诞生。这通常意味着有些部"
"分没有机会经过稳固坚实的设计。虽然我们一直在努力提高代码质量，但这并不意味着"
"项目能没有技术负债。有些地方仍可能会过于复杂而让人感到困惑。一旦你发现这种情形"
"的存在，都非常欢迎你给我们提出 PR！:-)"
```

**Review by archibate — COMMENTED** [review]
Thank for the hard work! I believe `contributor_guide.rst` is too long and unfriendly to taichi-docs-zh-cn... @yuanming-hu Can we refactor this by breaking it down? I can hardly track the diff here.

**Review by archibate — APPROVED** [review]
Thanks! I didn't read carefully word-to-word, but I think I could trust your translation :heart:

**Comment by ArkhamWJZ** [comment]
> Thanks! I didn't read carefully word-to-word, but I think I could trust your translation ❤️

That's a huge amount of trust 😆 I will try my best in the future work 🌈

**Pull request merged**